src/components/Project/ImageAnalysis/ImageAnalyzer.vue

```diff
@@ -121,11 +128,49 @@ function handleAnalysisOutput(response, action, action_callback){
       </div>
     </v-sheet>
   </v-dialog>
+  <v-dialog
```

Comment:
Unfortunately this isn't the FITS header, this is just the archive image record.

The FITS header data would need to either come from the backend via the FITS file, or the archive API. I think combining the headers from all HDUs in a FITS file and sending it alongside the large image URL would work.

You can also query it from the archive directly, e.g. `GET /frames/<id>/headers` but that wouldn't work if we decide to support images from other sources, like other archives.

Reply:
Oops, it had some similar data so I thought it was the header but I can see this is archive specific now.

Reply:
This is getting into multiple archive support, I think there should be some planning around this before I just come up with a temporary solution that may not work when we actually implement it

Reply:
Right - agreed that we will need to plan for multiple archive support, but returning the header data from the backend will work regardless of what the data source is - we just need the FITS file and we can serve it.

Reply:
I think in that case then I need to make a new header endpoint because we arn't getting the image URLs from the backend *I think* The way Jon wrote the image caching is that it goes to the archive directly?